crates/symbolicator/src/endpoints/mod.rs

```diff
@@ -34,7 +34,7 @@ pub fn create_app(service: Service) -> App {
         .route("/proxy/:path", get(proxy).head(proxy))
         .route("/requests/:request_id", get(requests))
         .route("/symbolicate", post(symbolicate))
-        .route("/minidump", post(minidump))
+        .route("/minidump", post(handle_minidump))
```

Comment:
I'm confused by this.  was this pushed to the right PR?

Reply:
I was a bit surprised myself. While rebasing on master, the compiler threw an error at me, so I just fixed that :-D

Reply:
Oh okay, turns out the problem was that we added a `minidump` crate which was ambiguous with the local `minidump` sub-module. This only happened since I rebased on master which has the axum refactor.

Reply:
ah, makes sense!